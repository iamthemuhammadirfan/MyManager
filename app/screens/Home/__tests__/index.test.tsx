import React from "react";
import {render} from "@testing-library/react-native";

import HomeScreen from "screens/Home";

describe("Home Screen", () => {
  it("should matches with the snap shoots", () => {
    const tree = render(<HomeScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("should have components in it", () => {
    const {getByTestId} = render(<HomeScreen />);
    expect(getByTestId("icon")).toBeDefined();
  });
});
