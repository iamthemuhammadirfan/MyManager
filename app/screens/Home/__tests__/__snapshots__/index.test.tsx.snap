// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Screen should matches with the snap shoots 1`] = `
<RCTSafeAreaView
  style={
    [
      {
        "backgroundColor": "red",
        "flex": 1,
      },
      undefined,
    ]
  }
  testID="container"
>
  <FontAwesome
    color="#ffffff"
    name="rocket"
    size={30}
    testID="icon"
  />
</RCTSafeAreaView>
`;
